share/ci/install/requirements_txt_modifier.py

```diff
@@ -53,21 +53,27 @@ def cs(text: str, color: str):
 
 if __name__ == "__main__":
     if len(sys.argv) < 3:
-        print("Set path to requirements.txt as first argument and output path as" " second argument.")
-        exit(1)
+        print("Set the path of the input requirements.txt as the second last argument "
+              "and the path of the output requirements.txt as the last argument.")
+        print("Set the name of the environment variables to ignore "
+              "before the path of the input requirements.txt")
+        sys.exit(1)
```

Comment:
Why do you use `sys.exit(1)` here? Wouldn't an exception have the same effect but be more specific and flexible? (Same at the bottom, of course.)

Reply:
No, you call a normal application without required arguments, it say you that the arguments are missing. But it will not print the call stack and other technical information.

Reply:
Yes, that's okay.